content/en/tracing/setup_overview/proxy_setup/_index.md

```diff
@@ -124,11 +124,26 @@ static_resources:
     traffic_direction: OUTBOUND
     filter_chains:
     - filters:
-      - name: envoy.http_connection_manager
+      - name: envoy.filters.network.http_connection_manager
         typed_config:
-          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
           generate_request_id: true
-          tracing: {}
+          tracing:
+          # Use the datadog tracer
+            provider:
+              name: envoy.tracers.datadog
+              typed_config:
+                "@type": type.googleapis.com/envoy.config.trace.v3.DatadogConfig
+                collector_cluster: datadog_agent   # matched against the named cluster
+                service_name: envoy-v1.19          # user-defined service name
+          access_log:
```

Comment:
This part is about integrating tracing and logs, is optional and could be done in different ways (eg: a JSON-based format). Not sure where that should be explained, but something to think about.
Also I think the logs pipeline for envoy needs changing to understand the additional trace and parent IDs in this format.